app/components/base/editor/editor.component.ts

```diff
@@ -49,6 +50,7 @@ export class EditorComponent implements ControlValueAccessor, AfterViewInit, OnC
     public placeholder: string;
     private _value = "";
     private _sub: Subscription;
+    private _erd: any;
```

Comment:
what is an erd? :) maybe call it _resizeDetector, i am a firm believer of descriptive variable names. Particularly helpful for junior devs.

Reply:
yeah was the same name in the heatmap and copied it from there. But can change it